src/dashboards/Chrono/styles.scss

```diff
@@ -3,15 +3,103 @@
 .chrono {
     &__tiles {
         display: flex;
-        flex-direction: row;
+        flex-direction: column;
```

Comment:
Hvorfor column her? Skal ikke tilsa fortsatt ligge horisontalt? 🤔

Reply:
Ja, men merkelig nok oppfører den seg omvendt. Column blir som row 🤔

Reply:
Hahah, what? 🤯 Ikke en stor greie, men har du noen insight om det her @draperunner ?

Reply:
Hm, nei, ikkje uten ein grundigare kikk!

Reply:
Den stylinga er på en div som wrapper `<ResponsiveReactGridLayout>`, viss tiles inni gridet da skal oppføre seg som inline-block's gir det mening for meg?